UnityAds/src/main/java/com/mopub/mobileads/UnityRewardedVideo.java

```diff
@@ -161,28 +162,16 @@ public void show() {
             return;
         }
 
-        if (UnityAds.isReady(mPlacementId)) {
-            // Lets Unity Ads know when ads succeeds to show
-            MediationMetaData metadata = new MediationMetaData(mLauncherActivity);
-            metadata.setOrdinal(++impressionOrdinal);
-            metadata.commit();
-
-            UnityAds.show(mLauncherActivity, mPlacementId);
-        } else {
-            // Lets Unity Ads know when ads fail to show
-            MediationMetaData metadata = new MediationMetaData(mLauncherActivity);
-            metadata.setMissedImpressionOrdinal(++missedImpressionOrdinal);
-            metadata.commit();
+        if (mLoadedPlacementId ==  null) {
+            UnityAds.show(mLauncherActivity, mPlacementId, mUnityShowListener);
+            return;
+        }
 
-            MoPubLog.log(CUSTOM, ADAPTER_NAME, "Attempted to show Unity rewarded video before it was available.");
-            MoPubLog.log(SHOW_FAILED, ADAPTER_NAME,
-                    MoPubErrorCode.VIDEO_PLAYBACK_ERROR.getIntCode(),
-                    MoPubErrorCode.VIDEO_PLAYBACK_ERROR);
+        MediationMetaData metadata = new MediationMetaData(mLauncherActivity);
+        metadata.setOrdinal(++impressionOrdinal);
+        metadata.commit();
```

Comment:
is this data still valuable.  I removed the missed impression case because it was no longer accurate. @andrewkonecny

Reply:
Yes, `impressionOrdinal` is still collected and sent to auction for every request. Removal of missed impression ordinal is fine since we will have the failed context from native+webview

Reply:
Ignore my previous comment about that then :P